### DISCUSSION TITLE
Split tabs with the last visited tab instead of the tab below

### DISCUSSION THREAD

**Discussion by tryfick64** [zen-browser/desktop#1572] [Ideas]
Hi,
I have a suggestion for improving the tab management in the browser. When opening a split tab (via shortcut, or split taps button when Rmb, when no other tabs selected), it would be more intuitive if the new tab split with the last visited tab, rather than the tab directly under the current one.
Additionally, it would be fantastic if two split tabs could be grouped into one, similar to how it's done in the Arc Browser. This would help because, visually, it's not always clear that the tabs are split when you're just looking at them. 
![изображение](https://github.com/user-attachments/assets/a8521840-55dd-4933-9bf8-f78eceff6507)
![изображение](https://github.com/user-attachments/assets/5458a523-0a13-48cd-810a-68c97a9b8bc4)
_ps. used AI to write this because my English isn't very good)_

**Comment by JSGRD22** [comment]
I think it would be better if the split tab button/shortcut instead left a blank space with a menu to select which tab to split, when only 1 tab is selected.
Regarding the visual tab grouping, I agree with you, but trying to split with pinned tabs would be a bit weird. Still not quite sure how that could be implemented. Perhaps pinned tabs could be given an overhaul where clicking on them opens a new tab with the site instead of opening it inside the pin, this would be a general QoL improvement. (I think the Arc equivalent of bookmarks functions this way)
